### DISCUSSION TITLE
in the CSV templates, the payer and plan names can't include |

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/hospital-price-transparency#75] [Q&A] [answered]
Given the use of pipe characters as delimiters in the column headers in the CSV format and the inclusion of the payer and plan names in the header, parsing will break if a pipe character ("|") is used in the payer or plan name.

Is this restriction in the specification?  

We see plan names that include pipes, but so far only in files using the JSON template e.g. "Medica" as payer and "Medicare|All Plans" as the plan.

**Comment by CarlsonErin** [comment, marked as answer]
Neil,

Thanks for bringing this to our attention, and keep them coming!
